src/page-modules/contact/validation/commonFieldValidator.ts

```diff
@@ -6,17 +6,39 @@ export const commonFieldValidator = (context: any) => {
   const inputFieldRules = [
     {
       field: 'firstName',
```

Comment:
Would it be better to not have validation per input field, but rather one for "contact information"? If form type is one of the ones listed in this form, we require first name, last name and email if the user wants to be contacted.